src/Products/Annotation/Controllers/AnnotationApiController.cs

```diff
@@ -12,168 +21,180 @@
 using GroupDocs.Annotation.MVC.Products.Annotation.Util.Directory;
 using GroupDocs.Annotation.MVC.Products.Common.Entity.Web;
 using GroupDocs.Annotation.MVC.Products.Common.Resources;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Net;
-using System.Net.Http;
-using System.Net.Http.Headers;
-using System.Web;
-using System.Web.Http;
-using System.Web.Http.Cors;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Controllers
 {
     /// <summary>
-    /// SignatureApiController
+    /// SignatureApiController.
     /// </summary>
     [EnableCors(origins: "*", headers: "*", methods: "*")]
     public class AnnotationApiController : ApiController
     {
-        private static Common.Config.GlobalConfiguration GlobalConfiguration;
-        private List<string> SupportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".gif", ".emf", ".wmf", ".dwg", ".dicom", ".djvu" };
-        private List<string> SupportedDiagrammFormats = new List<string>() { ".vsd", ".vdx", ".vss", ".vsx", ".vst", ".vtx", ".vsdx", ".vdw", ".vstx", ".vssx" };
-        private static AnnotationImageHandler AnnotationImageHandler;
-        private DirectoryUtils DirectoryUtils;
+        private static Common.Config.GlobalConfiguration globalConfiguration;
+        private static AnnotationImageHandler annotationImageHandler;
+        private readonly List<string> supportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".gif", ".emf", ".wmf", ".dwg", ".dicom", ".djvu" };
+        private readonly List<string> supportedDiagrammFormats = new List<string>() { ".vsd", ".vdx", ".vss", ".vsx", ".vst", ".vtx", ".vsdx", ".vdw", ".vstx", ".vssx" };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5681088)